recipes/jupyter-widget-pivot-table/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.1.3" %}
+{% set md5 = "7eebff0d6da4fed51cfb143c4dcd0011" %}
+
+package:
+  name: jupyter-widget-pivot-table
+  version: {{ version }}
+
+source:
+  fn: jupyter-widget-pivot-table-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyter_widget_pivot_table/jupyter_widget_pivot_table-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
```

Comment:
It looks like this requires `pip` to build

Reply:
Thanks, I am currently in the process of changing the underlying PyPI package to avoid having to use pip in the build process (it's actually a useless dependency)

Reply:
You can also package from github if you prefer.

Reply:
OK thanks ! (Actually the problem is not publishing on PyPI, but rather changing the Python code itself.)